### PR TITLE
User's name and email address is displayed on the AccountDetails screen.

### PR DESCRIPTION
**Pull request by tahaafzal5**
Hey can you look at this? Since we don't have the user's profile picture on Parse rn and also no rides, I have not populated those fields on the AccountDetails screen yet. But the name and email address show up for the user currently logged in.

### CODE REVIEW

**Comment by tahaafzal5** [comment]
So I added the default image to be shown. However, it does not automatically update the ImageView once a new image is set. It updates only when you log out and log in again. Looks like it might have something to do with how Parse deals with Images or even with Cache. 

I created a new issue for this that we can look into after the demo.

File: rideshare/Controllers/AccountDetailsViewController.swift [hunk 1]
```diff
@@ -25,6 +31,27 @@ class AccountDetailsViewController: UIViewController, UITableViewDataSource, UIT
         if let index = self.tableView.indexPathForSelectedRow {
             tableView.deselectRow(at: index, animated: animated)
         }
+
```
Comment (nashirj) [inline rideshare/Controllers/AccountDetailsViewController.swift:34]: You can simplify 35-45 by just saying
``` Swift
if (user?["profilePicture"] != nil) {
    let imageFile = user?["profilePicture"] as! PFFileObject
    let urlString = imageFile.url!
    let url = URL(string: urlString)!

    profilePictureImage.af.setImage(withURL: url)
//    profilePictureImage.setNeedsDisplay()
}
else {
    profilePictureImage.image = UIImage(systemName: "person")
}
```

**Review by nashirj — APPROVED** [review]
Looks good to me! You can make the change I suggested if you want, but I'm fine with merging this in as is

**Comment by tahaafzal5** [comment]
Yeah this is better, so I changed it. Should be good to be merged in!

**Pull request merged**